.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         - /dtl_candidate_.*/
     parallelism: 2
     docker:
-      - image: wjordan/code-dot-org:0.5
+      - image: jmkulwik/code-dot-org:0.6
```

Comment:
revert temporary changes here?

Reply:
This one will be a permanent change

Reply:
I'm not super familiar with how docker naming schemes work, but is there a specific reason this is namespaced to an individual rather than to the organization?

Reply:
In this case, I think it's just because we don't have an organization on docker hub. @uponthesun or @wjordan might have more details.